api/handlers/container/create_test.go

```diff
@@ -439,17 +439,19 @@ var _ = Describe("Container Create API ", func() {
 			Expect(rr.Body).Should(MatchJSON(jsonResponse))
 		})
 
-		It("should set CapDrop option", func() {
+		It("should set CapDrop and GroupAdd option", func() {
```

Comment:
Why not add a separate test for GroupAdd?

Reply:
Test description needs to be updated

Reply:
This still needs to the description to be reverted.